src/App.css

```diff
@@ -18,3 +37,207 @@ nav,
   column-gap: 24px;
   align-items: center;
 }
+
+.links a {
+  opacity: 0.5;
+}
+
+.links a.active {
+  opacity: 1;
+}
+
+.login {
+  border: 0.1px solid #dcd7d7;
+  border-radius: 50%;
+  padding: 0.875rem;
+  width: 16px;
+  height: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: var(--azure);
+  margin-left: auto;
+  cursor: pointer;
+}
+
+/* booklist */
+.book-list {
+  display: flex;
```

Comment:
- [ ] (Optional) It would be nice to create a CSS selector for declarations that are used throughout your stylesheet. For example, in your code, you used `display: flex;` regularly, you could create a CSS selector and have this declaration in it like this:

```
.display-flex {
  display: flex;
}
```
Then you can use the seletor as a `className` for the specific `jsx` element.